src/os/bluestore/BlueStore.cc

```diff
@@ -9756,7 +9756,7 @@ int BlueStore::_do_read(
 
   int64_t num_ios = length;
   if (ioc.has_pending_aios()) {
-    num_ios = -ioc.get_num_ios();
+    num_ios = ioc.get_num_ios();
```

Comment:
```suggestion
    num_ios -= ioc.get_num_ios();
```

Reply:
in fact, num_ios is mean aio ios. So num_ios= ioc.get_num_ios is ok. But for non-aio, the value is strange.

Reply:
hmm, i think a better alternate fix would be initiating num_ios at 0 + this patch since it's mainly used for measuring the reading latencies - if ioc ends up with no pending aios, it means all required data are fulfilled from the local cache and no aio is issued onto disk at all

Reply:
maybe no not use libaio.

Reply:
@xiexingguo . how about use  blobs2read.size() for initial value?

Reply:
yeah, sounds reasonable to me